src/server/store.rs

```diff
@@ -21,6 +21,26 @@ fn url_to_key_val(url: &lsp::Url) -> (String, String) {
     (parent, filename.into())
 }
 
+fn get_analyzer() -> Result<flux::semantic::Analyzer<'static, &'static flux::semantic::import::Packages>, LspError> {
+    match flux::new_semantic_analyzer(
+        flux::semantic::AnalyzerConfig {
+            // Explicitly disable the AST and Semantic checks.
+            // We do not care if the code is syntactically or semantically correct as this may be
+            // partially written code.
+            skip_checks: true,
```

Comment:
`skip_checks: true` were there before but I don't think it should be as it will cause us to not report the errors that are skipped. Since the `analuyze*` functions now continue running all checks even after errors occur and they return a `SalvageResult` the correct way to ignore errors would be pick the `Package `out from the `Salvage` type when errors occurs. https://github.com/influxdata/flux/pull/4710

Reply:
Since this was a "nits" branch, I didn't think it was a big deal to address all the nits from review. Turns out that there were no tests for `get_package_errors`, so I added those, and then removed the `skip_checks` and just took the default. We were already extracting the `Package` out of the result anyway, so it shouldn't really be an issue. If a default were to change in `flux` now, we'll have tests to know that expected behavior is changed. Thanks for the heads up.